front/src/applications/rollingStockEditor/components/RollingStockEditorForm.tsx

```diff
@@ -53,33 +64,38 @@ const RollingStockEditorInput = ({
   const { t } = useTranslation('rollingStockEditor');
   return (
     <div
-      className={`d-flex flex-row${
-        position ? '' : '-reverse'
+      className={`d-flex ${
+        !position && 'rollingstock-editor-input'
       } justify-content-between align-items-center mb-2`}
     >
-      <label className={`text-primary col-8 ${position ? 'mr-2' : 'ml-2'} my-0`} htmlFor={label}>
+      <label
+        className={`text-primary col-8 ${position && 'rollingstock-editor-input-label'} my-0`}
+        htmlFor={label}
+      >
         {error && (
           <div className="text-yellow">
             <BiErrorCircle />
           </div>
         )}
-        {t(label)}
+        <span className={!position ? 'right-side' : ''}>{t(label)}</span>
       </label>
       <input
         id={label}
-        className={`form-control-sm bg-white px-2 ${!position ? 'col-3' : 'col-4'}`}
+        className={`form-control-sm bg-white px-2 ${
+          !position && label.includes('rolling_resistance') ? 'col-3' : 'col-4'
+        }`}
         {...register(label)}
       />
-      {!position && (
-        <span className="col-1">
-          {label.substring(label.length - 1, label.length).toLocaleLowerCase()}:{' '}
-        </span>
+      {!position && label.includes('rolling_resistance') && (
```

Comment:
It won't work anymore the day the labels changes. Use a boolean: isRollingResistance in your RollingStockEditorElementsProps for instance

Reply:
ok

Reply:
done